packages/calcite-components/src/components/chip/chip.scss

```diff
@@ -1,359 +1,359 @@
+      }
 
-  .multiple {
-    .select-icon {
-      --calcite-internal-chip-padding-start: var(--calcite-spacing-xxs);
-      --calcite-internal-chip-padding-end: var(--calcite-spacing-xxs);
+      &:has(.chip-icon),
+      &.text--slotted,
+      &.closable {
+        --calcite-internal-chip-image-space-x-end: 0.5rem /* 8px */;
+      }
     }
 
-    &.image--slotted {
-      .select-icon {
-        --calcite-internal-chip-padding-start: var(--calcite-spacing-xs);
-        --calcite-internal-chip-padding-end: var(--calcite-spacing-md);
+    &.selectable.single:not(.is-circle) {
+      &.image--slotted {
+        --calcite-internal-chip-container-space-x-start: 0.375rem /* 6px */;
+      }
+      &.selected {
+        --calcite-internal-chip-select-space-x-end: 0.25rem /* 4px */;
+        --calcite-internal-chip-select-space-x-start: 0;
+
+        &.image--slotted {
+          --calcite-internal-chip-select-space-x-end: 0.375rem /* 6px */;
+          --calcite-internal-chip-select-space-x-start: 0.5rem /* 8px */;
+        }
       }
     }
-  }
 
-  .container {
-    --calcite-internal-chip-padding-start: var(--calcite-spacing-sm);
-    --calcite-internal-chip-padding-end: var(--calcite-spacing-sm);
+    &.multiple:not(.is-circle) {
+      --calcite-internal-chip-container-space-x-start: 0.5rem /* 8px */;
 
-    &.image--slotted {
-      --calcite-internal-chip-padding-start: var(--calcite-spacing-xs);
-      --calcite-internal-chip-padding-end: var(--calcite-spacing-sm);
+      --calcite-internal-chip-select-space-x-end: 0.25rem /* 4px */;
+      --calcite-internal-chip-select-space-x-start: 0.25rem /* 4px */;
 
-      &.closable {
-        --calcite-internal-chip-padding-start: var(--calcite-spacing-xs);
-        --calcite-internal-chip-padding-end: var(--calcite-spacing-xs);
+      &.image--slotted {
+        --calcite-internal-chip-select-space-x-end: 0.75rem /* 12px */;
       }
     }
-  }
-
-  .container:not(.is-circle):not(.multiple) {
-    &:not(.image-slotted) .select-icon.select-icon--active {
-      padding-inline-start: 0;
-      padding-inline-end: var(--calcite-spacing-xxs);
-    }
 
-    &.image--slotted .select-icon.select-icon--active {
-      padding-inline-start: 0;
-      padding-inline-end: var(--calcite-spacing-md);
+    &.closable:not(.is-circle) {
+      --calcite-internal-chip-container-space-x-end: 0.375rem /* 6px */;
     }
   }
 }
 
-:host([scale="l"][closable]) .container {
-  --calcite-internal-chip-padding-start: var(--calcite-spacing-sm);
-  --calcite-internal-chip-padding-end: var(--calcite-spacing-xs);
-}
-
-:host {
-  @apply inline-flex
-    cursor-default
-    rounded-full;
-}
-
+// Sub-element styles
 .container {
   @apply inline-flex
   h-full
   max-w-full
   items-center
-  rounded-full
   focus-base
   justify-center
-  border
-  border-solid
-  border-color-1
   box-border
   font-medium;
 
+  border-radius: var(--calcite-chip-corner-radius, 9999px);
```

Comment:
Sidebar: we still need a semantic token for this fallback, right?

Reply:
yes this will be coming in future updates